apps/users/serializers.py

```diff
@@ -116,4 +119,83 @@ def update(self, instance, validated_data):
             instance.profile_picture_url = profile_picture_url
             instance.save()
         return super().update(instance, validated_data)
-    
\ No newline at end of file
+
+#=================================================================================================
+#change known Password serializer
+class UserChangePasswordSerializer(serializers.Serializer):
+    old_password = serializers.CharField(max_length=255, style={'input_type': 'password'}, write_only=True)
+    password = serializers.CharField(max_length=255, style={'input_type':'password'}, write_only=True)
+    confirm_password = serializers.CharField(max_length=255, style={'input_type':'password'}, write_only=True)
+
+    class Meta:
+        fields=['old_password', 'password', 'confirm_password']    
+    def validate(self, attrs):
+        old_password = attrs.get('old_password')
+        password = attrs.get('password')
+        confirm_password = attrs.get('confirm_password')
+        user = self.context.get('user')
+
+        # Validate old password
+        if not user.check_password(old_password):
+            raise serializers.ValidationError({"old_password": "Old password is incorrect"})
+
+        if password != confirm_password:
+            raise serializers.ValidationError("Password and confirm password doesn't match")
+        
+        user.set_password(password)
+        user.save()
+        return attrs
+
+#====================================================================================================
+#forgot passswd serializer
+class SendPasswordResetEmailSerializer(serializers.Serializer):
+    email = serializers.EmailField(max_length=255)
+    class Meta:
+        fields = ['email']
+    def validate(self, attrs):
+        email = attrs.get('email')
+        if User.objects.filter(email=email).exists():
+            #if exists get user object from DB
+            user = User.objects.get(email=email)
+            uid = urlsafe_base64_encode(force_bytes(user.user_id))
+            token = CustomPasswordResetTokenGenerator().make_token(user)
```

Comment:
This http://127.0.0.1:8000 should come as baseurl from util_variables and we need to change it to VPS host while deplloying, otherwise it fail

Reply:
added baseurl = 'http://195.35.20.172:8000' in utils_variables.py file and importing baseurl in serializers.py files.